app/searchfunctions.py

```diff
@@ -179,18 +179,18 @@ def iat_employee_table_headers():
 
     return add_headers
 
-def iat_employee_record_table(employee_records_json):
+def iat_employee_record_table(employee_records_json, remove_html = False):
     add_employees = []
     for employees in employee_records_json:
         add_employees.append({'tds': [
             {
                 'value': employees['unique_role_id']
             },
             {
-                'value': acc_generation(str(employees['ons_email_address']))
+                'value': acc_generation(str(employees['ons_email_address'])) if  remove_html == False else str(employees['ons_email_address'])
```

Comment:
You don't need to compare booleans to booleans. `if not remove_html else` will suffice.

Reply:
Changed